support/scripts/patch-rpath.py

```
@@ -0,0 +1,116 @@
+			for cmd in binary.commands
+			if cmd.command
+			in (
+				lief.MachO.LoadCommand.TYPE.LOAD_DYLIB,
+				lief.MachO.LoadCommand.TYPE.LOAD_WEAK_DYLIB,
+			)
+		]
+	logger.info(f'Needed libraries after patching: {needed}')
+
+	if codesign and binary.format == lief.Binary.FORMATS.MACHO:
+		logger.info(f'Code signing Mach-O binary: {path}')
+		subprocess.run(['rcodesign', 'sign', str(path)], check=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)